x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -587,6 +587,9 @@ export class TaskRunner<
       ),
       schedule: resolveErr<IntervalSchedule | undefined, Error>(schedule, (error) => {
         if (isAlertSavedObjectNotFoundError(error, alertId)) {
+          this.logger.warn(
+            `Unable to execute rule "${alertId}" because ${error.message} - this rule will not be rescheduled. To restart rule execution, try disabling and re-enabling this rule.`
```

Comment:
Since this message is so actionable, I feel like we should include more info so we can make it as easy as possible for the user to find the alert to fix it.  Probably means adding the Kibana space and rule name.  Rule type would be interesting for diagnostic purposes / telemetry, but probably doesn't help a user that much - I would assume the rule name would provide most of the context to find the rule in Kibana ...

Reply:
So, this might be a little tricky since in this context, we only have access to the `ruleId` and presumably, since we're seeing the `Saved object not found` error, we have been unable to retrieve the saved object that would give us the `spaceId` and the `ruleName`.

Reply:
heh, right!  bummer!

I think we have seen this message when we get transient network problems, in which case we might have gotten the alert SO at the some point, then failed later when the alert ran.  In which case, some piece of code _had_ that info.  How we'd keep track on that seems ... difficult.  Ah well.